### PR TITLE
[tensorflow] Speculative fix to get at least one successful build.

### PR DESCRIPTION
**Pull request by Dor1s**
//cc @mihaimaruseac

### CODE REVIEW

File: projects/tensorflow/project.yaml [hunk 1]
```diff
@@ -3,3 +3,9 @@ primary_contact: "mihaimaruseac@google.com"
 auto_ccs:
  - "dga@google.com"
  - "frankchn@google.com"
+
```
Comment (inferno-chromium) [inline projects/tensorflow/project.yaml]: no need of empty line, same before sanitizers.

File: infra/gcb/build_project.py [hunk 2]
```diff
@@ -20,7 +20,7 @@
 from oauth2client.service_account import ServiceAccountCredentials
 from googleapiclient.discovery import build
 
-BUILD_TIMEOUT = 10 * 60 * 60
+BUILD_TIMEOUT = 20 * 60 * 60
```
Comment (inferno-chromium) [inline infra/gcb/build_project.py]: maybe even 30 minutes is ok ?

Comment (mihaimaruseac) [reply infra/gcb/build_project.py]: The build takes around 2 hours on local docker container :(

Comment (Dor1s) [reply infra/gcb/build_project.py]: These are hours :)

File: projects/tensorflow/project.yaml (hunk 1 above)
Comment (Dor1s) [reply projects/tensorflow/project.yaml]: Done!

File: infra/gcb/build_project.py (hunk 2 above)
Comment (Dor1s) [reply infra/gcb/build_project.py]: Yeah, but we build several configurations, and the cloud builder is likely a bit less powerful than our desktops :)

Just give me a second to double check something in the logs. I think I won't increase this timeout limit, actually.

Comment (Dor1s) [reply infra/gcb/build_project.py]: Yeah, remove this. Let's see if we can build a single configuration and how long it takes. Then we'll think what we should do next.

**Comment by Dor1s** [comment]
The build succeeded, though it took 7 hours to be produced. We're using the most powerful Cloud Builder machines, so there isn't much we can do to optimize the things on our end. We probably can bump build timeout from 10hr to e.g. 20hr, that would allow us to enable one or two more configs (e.g. UBSan and MSan), but I'm not sure if we should go that way. @mihaimaruseac @inferno-chromium @oliverchang what do you guys think?

**Comment by mihaimaruseac** [comment]
Let's not go that way at the moment, I'll be working on speeding up build / reducing amount of stuff that needs to be compiled in the next quarter and we'll revise then.

Thank you

**Comment by inferno-chromium** [comment]
@mihaimaruseac - thanks!

**Comment by Dor1s** [comment]
We've had a successful build over the past week, but I don't see any stats / logs on CF. @oliverchang are the build archives too large?

**Comment by mihaimaruseac** [comment]
Might also be because coverage build is failing, I'm looking into it now

**Comment by Dor1s** [comment]
No, coverage build failure won't be an issue here, it doesn't affect fuzzing.

**Comment by oliverchang** [comment]
FYI we were seeing some issues with disk space, as there are 16 targets at 1.5GB each. I've increased the disk space for tensorflow bots to see if this fixes it, but if the size of the binaries could be reduced that would be awesome.

**Comment by mihaimaruseac** [comment]
Reducing size of binaries will take a while. But I could skip a few of the fuzzers for now

**Pull request merged**